docs/manual.rst

```diff
@@ -1,19 +1,28 @@
-
 =======================
 Running pytest-workflow
 =======================
-Run ``pytest`` from an environment with pytest-workflow installed.
+Run ``pytest`` from an environment with pytest-workflow installed or
+``python3 -m pytest`` if using a system-wide or user-wide installation.
 Pytest will automatically gather files in the ``tests`` directory starting with
 ``test`` and ending in ``.yaml`` or ``.yml``.
 
 The workflows are run automatically. Each workflow gets its own temporary
 directory to run. These directories are cleaned up after the tests are
 completed. If you wish to inspect the output of a failing workflow you can use
-the ``--keep-workflow-wd`` flag to disable cleanup.
+the ``--keep-workflow-wd`` flag to disable cleanup. This will also make sure
+the logs of the pipeline are saved in the temporary directory. The
+``--keep-workflow-wd`` flag is highly recommended when debugging pipelines.
 
 If you wish to change the temporary directory in which the workflows are run
 use ``--basetemp <dir>`` to change pytest's base temp directory.
 
+.. container:: warning
+
+  WARNING: When a directory is passed to ``--basetemp`` some of the directory
+  contents will be deleted if they have the same name as any of the workflows
```

Comment:
This could use some extra clarity. Exactly which contents will be deleted, and where? I _think_ it means files in the temporary directory having the exact same filename as one of the workflow names will be deleted, but I could be reading this wrong.

Reply:
Additionally, the current codes entirely removes the entire directory if it already exists.